doc/conf.py

```diff
@@ -96,6 +103,15 @@
 
 autodoc_docstring_signature = True
 
+
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+   # path to your examples scripts
+    'examples_dirs' : '../examples',
+    # path where to save gallery generated examples
+    'gallery_dirs'  : 'auto_examples'}
```

Comment:
It's not that trivial, because matplotlib somehow does I don't know what in the plot directive that makes this a pain in the butt to change. Which is why I asked for feedback a month ago.